cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -244,17 +244,33 @@ public void upgrade(String tableName, Properties properties) throws Exception {
     }
   }
 
-  private void drop(byte[] tableName) throws IOException {
+  private void drop(TableId tableId) throws IOException {
     HBaseAdmin admin = getHBaseAdmin();
-    if (admin.tableExists(tableName)) {
-      admin.disableTable(tableName);
-      admin.deleteTable(tableName);
+    if (tableUtil.tableExists(admin, tableId)) {
+      tableUtil.disableTable(admin, tableId);
+      tableUtil.deleteTable(admin, tableId);
     }
   }
 
+  public TableId getConfigTableId(QueueName queueName) {
+    return getConfigTableId(queueName.getFirstComponent());
+  }
+
+  public TableId getConfigTableId(String namespace) {
+    return TableId.from(getConfigTableName(namespace));
+  }
+
+  public TableId getDataTableId(QueueName queueName) {
```

Comment:
any way to unify this and the method below? Use overloading so that their names are the same (and - hopefully - one calls the other?)